clients/src/test/java/org/apache/kafka/clients/consumer/internals/OffsetsRequestManagerTest.java

```diff
@@ -131,13 +131,11 @@ public void testListOffsetsRequest_Success() throws ExecutionException, Interrup
                 ListOffsetsRequest.EARLIEST_TIMESTAMP);
 
         mockSuccessfulRequest(Collections.singletonMap(TEST_PARTITION_1, LEADER_1));
-        CompletableFuture<Map<TopicPartition, OffsetAndTimestamp>> result = requestManager.fetchOffsets(
-                timestampsToSearch,
-                false);
+        CompletableFuture<Map<TopicPartition, Long>> result = requestManager.beginningOrEndOffset(timestampsToSearch);
         assertEquals(1, requestManager.requestsToSend());
         assertEquals(0, requestManager.requestsToRetry());
 
-        Map<TopicPartition, OffsetAndTimestamp> expectedOffsets = Collections.singletonMap(TEST_PARTITION_1, new OffsetAndTimestamp(5L, 1L));
+        Map<TopicPartition, Long> expectedOffsets = Collections.singletonMap(TEST_PARTITION_1, 5L);
```

Comment:
Note: A lot of mock results are converted to Long because most tests have the `requireTimestamps` field marked false.